modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/config/ApplicationConfiguration.java

```diff
@@ -46,4 +49,15 @@ public PasswordEncoder passwordEncoder() {
 
         return new DelegatingPasswordEncoder(encodingId, encoders);
     }
+
+    /**
+     * @return Application event multicaster.
+     */
+    @Bean(name = "applicationEventMulticaster")
+    public ApplicationEventMulticaster simpleApplicationEventMulticaster() {
+        SimpleApplicationEventMulticaster evtMulticaster = new SimpleApplicationEventMulticaster();
+
+        evtMulticaster.setTaskExecutor(new SimpleAsyncTaskExecutor());
```

Comment:
How many threads this executor will create?

Reply:
Change on ThreadPoolTaskExecutor.